gradle build fails

**What version of EMX2 are you using (see footer)**
v8.189.9

**Describe the bug**
`FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tools:pyclient:buildPython'.
> Process 'command 'python3'' finished with non-zero exit value 1
`

**To Reproduce**
Steps to reproduce the behavior:
1. checkout master
2.  ./gradlew build -x test


**Expected behavior**
Projects gets build

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -82,7 +82,7 @@ jobs:
     - run:
         name: build, test, release, push docker
         command: |
-          ./gradlew test --no-daemon shadowJar dockerPush release helmPublishMainChart ci \
+          ./gradlew test --no-daemon shadowJar dockerPush release helmPublishMainChart buildPython ci \
           -Dsonar.login=${SONAR_TOKEN} -Dsonar.organization=molgenis -Dsonar.host.url=https://sonarcloud.io \
           -Dorg.ajoberstar.grgit.auth.username=${GITHUB_TOKEN} -Dorg.ajoberstar.grgit.auth.password
           export $( cat build/ci.properties | xargs )

--- a/tools/README.md
+++ b/tools/README.md
@@ -12,8 +12,6 @@ You can run within that folder
 or it make more sense to actually say
 ```python3 -m build```
 
-Note that ```gradle build``` will include python next to the java and javascript in the overall build.
-
 ## To add a new package you need to
 * add a folder with a valid python package structure
 * add your new project to molgenis-emx2/settings.gradle

--- a/tools/build.gradle
+++ b/tools/build.gradle
@@ -25,11 +25,6 @@ subprojects {
         inputs.file('README.md')
     }
 
-    //also run when overall build is run
-   task build {
-       finalizedBy "buildPython"
-    }
-
     task clean {
         doLast {
             delete 'dist'

